Makefile

```diff
@@ -1,13 +1,18 @@
 PROG=ytfzf
 PREFIX=/usr/local
 BINDIR=${PREFIX}/bin
+DOCDIR=${PREFIX}/share/doc/ytfzf
 MANDIR=${PREFIX}/share/man
 
-man:
+doc:
 	mkdir -p ${DESTDIR}${MANDIR}/man1
 	mkdir -p ${DESTDIR}${MANDIR}/man5
+	mkdir -p ${DESTDIR}${DOCDIR}
+	mkdir -p ${DESTDIR}/share/licenses
 	install docs/man/ytfzf.1 ${DESTDIR}${MANDIR}/man1
 	install docs/man/ytfzf.5 ${DESTDIR}${MANDIR}/man5
+	install docs/conf.sh ${DESTDIR}${DOCDIR}
+	install LICENSE ${DESTDIR}/share/licenses
```

Comment:
I think this is meant to be `${DESTDIR}${PREFIX}/share/licenses`, currently it installs to `/share/licenses`